### PR TITLE
Improve the headings of the databases section of the reference manual

### PR DESCRIPTION
**Pull request by kwankyu**
This part of the reference manual is in poor state. We improve the look of the part. 

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/sage/databases/findstat.py [hunk 1]
```diff
@@ -1,38 +1,28 @@
...
-- Martin Rubey (2015): initial version.
-- Martin Rubey (2020): rewrite, adapt to new FindStat API
-
-The main entry points
----------------------
-.. csv-table::
-    :class: contentstable
-    :widths: 20, 40
-    :delim: |
+The main entry points to the database is
```
Comment (mantepse) [inline src/sage/databases/findstat.py]: "is" should be "are:", or, if you remove the two superfluous ":" in line 16 and 10, "are"

File: src/sage/databases/findstat.py [hunk 2]
```diff
@@ -1,38 +1,28 @@
...
 
-Fix the following three notions:
+We use the following three notions:
 
 - A *combinatorial collection* is a set `S` with interesting combinatorial properties,
 - a *combinatorial map* is a combinatorially interesting map `f: S \to S'` between combinatorial collections, and
 - a *combinatorial statistic* is a combinatorially interesting map `s: S \to \ZZ`.
 
-You can use the sage interface to FindStat to:
+You can use the Sage interface to:
```
Comment (mantepse) [inline src/sage/databases/findstat.py]: it should either be "the interface" or "the FindStat interface"

File: src/sage/databases/findstat.py [hunk 3]
```diff
@@ -1,38 +1,28 @@
 # -*- coding: utf-8 -*-
 r"""
-FindStat - the Combinatorial Statistic Finder.
+FindStat - the search engine for combinatorial statistics and maps
 
-The FindStat database can be found at::
+The Sage interface to the FindStat database is::
```
Comment (mantepse) [inline src/sage/databases/findstat.py]: I prefer the old wording, or alternatively "The interface to".  Nota bene, throughout the file, "Sage" is still in lower case, so using "Sage" here introduces a very minor inconsistency.

File: src/sage/databases/findstat.py (hunk 1 above)
Comment (kwankyu) [reply src/sage/databases/findstat.py]: Done, with "are", removing colons.

File: src/sage/databases/findstat.py (hunk 3 above)
Comment (kwankyu) [reply src/sage/databases/findstat.py]: Thanks for the input. Changed to "Sage" throughout the file.

File: src/sage/databases/findstat.py (hunk 2 above)
Comment (kwankyu) [reply src/sage/databases/findstat.py]: Done with the latter.

File: src/sage/databases/cubic_hecke_db.py [hunk 4]
```diff
@@ -31,10 +31,24 @@
 of linear forms on the cubic Hecke algebra on at most four strands
 satisfying the Markov trace condition for its cubic Hecke subalgebras.
 
+To use the database, you need to install the optional package by the Sage
+command ::
+
+    sage -i database_cubic_hecke
```
Comment (mkoeppe) [inline src/sage/databases/cubic_hecke_db.py:38]: Ideally, whenever a `sage -i` line is shown, there should also be a link to the SPKG page such as https://doc.sagemath.org/html/en/reference/spkg/database_cubic_hecke.html#spkg-database-cubic-hecke;
for some of the packages there are install instructions for equivalent system packages

Comment (kwankyu) [reply src/sage/databases/cubic_hecke_db.py:38]: Yes. Done.

**Comment by mkoeppe** [comment]
```
 [sagemath_doc_html-none] OSError: /sage/src/sage/databases/cunningham_tables.py:docstring of sage.databases.cunningham_tables:1: WARNING: undefined label: 'cunningham_tables'
 ```

**Comment by kwankyu** [comment]
Sorry! Fixed.

**Review by mkoeppe — APPROVED** [review]
LGTM. Thanks

**Comment by kwankyu** [comment]
Thanks. I added links to the optional packages for the remaining two(three) databases.

**Review by mkoeppe — APPROVED** [review]

**Comment by kwankyu** [comment]
Thanks for the review!

**Pull request merged**